docs/apps.md

```diff
@@ -326,37 +326,44 @@ Content-Type: application/vnd.api+json
 #### Advanced usage
 
 Two optional query parameters are available for an app update:
-- `PermissionsAcked`: (defaults to `true`)
-  - Tells that the user accepted the permissions/ToS. It is useful if there are
+-   `PermissionsAcked`: (defaults to `true`)
+      - Tells that the user accepted the permissions/ToS. It is useful if there are
     newer permissions or Terms Of Service and you want to be sure they were read
     or accepted. If set to `false`, the update will be blocked and the user will
-    be told that a new app version is available
-
-- `Source` (defaults to `SourceURL` installation parameter):
-  - Use a different source to update this app (e.g. to install a `beta` or `dev`
+    be told that a new app version is available.\
```

Comment:
[nit] Why do we have a `\` at the end of the line?

Reply:
I was not sure if the MKdocs markdown interpreter would add the carriage return, so I forced it. In the past, I noticed some differences between my markdown previews and the generated documentation.